src/extended_audio.c

```diff
@@ -0,0 +1,211 @@
+/*
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Deviation is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Deviation.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "common.h"
+#include "music.h"
+#include "config/tx.h"
+#include "config/model.h"
+#include "extended_audio.h"
+#include "stdlib.h"
+
+#ifndef EMULATOR
+#include <libopencm3/stm32/usart.h>
+#endif // EMULATOR
+
+#if HAS_EXTENDED_AUDIO
+
+static u32 audio_queue_time = 0;
+
+// Initialize UART for extended-audio
+void AUDIO_Init() {
+#ifdef BUILDTYPE_DEV
```

Comment:
You can omit #ifdef BUILDTYPE_DEV - in non dev builds printf is an empty macros so it's totally eliminated and is not present in the final binary at all.